File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoLayoutCard.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnGridRenderingMode;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutCard extends StudioDtoLayoutGrid
{
  @Nullable
  public StudioDtoLayoutCardFilter filter;

  @Nullable
  public MetaIdField groupByFieldId;

  @Nullable
  public Boolean hideBorders;

  @Nullable
  public StudioDtoLayoutCardItem item;

  @Nullable
  public Long numOfColumns;

  @Nullable
  public EnumDefnGridRenderingMode renderingMode;

  @Nullable
  public Boolean showSearchBar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutCard.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnGridRenderingMode
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.StudioDtoLayoutCardFilter
import com.neome.api.meta.base.dto.StudioDtoLayoutCardItem
import com.neome.api.meta.base.dto.StudioDtoLayoutGrid

interface StudioDtoLayoutCard : StudioDtoLayoutGrid
{
  val filter: StudioDtoLayoutCardFilter?
  val groupByFieldId: MetaIdField?
  val hideBorders: Boolean?
  val item: StudioDtoLayoutCardItem?
  val numOfColumns: Long?
  val renderingMode: EnumDefnGridRenderingMode?
  val showSearchBar: Boolean?
}